config/routes.rb

```diff
@@ -21,6 +21,8 @@
 
     get '/*route/folder_new', to: 'folders#new', as: :new_folder
     get '/*route/:binary_name', to: 'folders/binaries#show', as: :binary, format: true
+    post '/*route/:binary_name/like', to: 'folders/binaries/likes#create', as: :like_file, format: true
```

Comment:
@jk1dd you'd need to create a separate route and controller for the comment's likes. what is the `format: true` flag providing you with here?